.github/workflows/build.yaml

```diff
@@ -52,3 +52,66 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  wheel-build-pylibcugraph:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@main
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cugraph
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: pylibcugraph
+      package-dir: python/pylibcugraph
+
+      # Note that this approach to cloning repos obviates any modification to
+      # the CMake variables in get_cumlprims_mg.cmake since CMake will just use
+      # the clone as is.
+      extra-repo: rapidsai/cugraph-ops
+      extra-repo-sha: branch-23.02
+      extra-repo-deploy-key: CUGRAPH_OPS_SSH_PRIVATE_DEPLOY_KEY
+
+      skbuild-configure-options: "-DDETECT_CONDA_ENV=OFF -DCUGRAPH_BUILD_WHEELS=ON -DCPM_cugraph-ops_SOURCE=/project/cugraph-ops/"
+  wheel-publish-pylibcugraph:
+    needs: wheel-build-pylibcugraph
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-publish.yml@main
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cugraph
```

Comment:
we can remove this `repo` key from all of the `build.yaml` workflows (similarly to how I did for rmm [here](https://github.com/rapidsai/rmm/commit/7fb45d44c8ca4151581f5c57e413d1aea4b00f3f)).

This key was only ever used because we were using a `workflow_call` event to call this workflow from the https://github.com/rapidsai/actions/ repository for nightly builds.

Now that this file uses `workflow_dispatch` instead of `workflow_call`, this key isn't relevant anymore.

We should make this change for all jobs in this workflow and also all other wheel PRs.